assets/css/stripeStyle.css

```diff
@@ -132,3 +111,172 @@ input:focus {
 .hidden {
 	display: none;
 }
+
+
+
+form {
+	display: flex;
+	flex-wrap: wrap;
+	padding: 0.5rem;
+}
+
+form h2 {
+	width: 100%;
+	padding: 0.5rem;
+	color: #304FFE;
+}
+
+.form-field-container {
+	width: 50%;
+	padding: 0.5rem;
+}
+
+.form-field-container label {
+	font: 600 12px/17px 'Open Sans';
+	color: #616161;
+}
+
+
+.form-field-container input,
+.form-field-container select {
+	margin-top: 1rem;
+}
+
+.form-field-container input.prefix-input {
+	margin-top: 0;
+}
+
+.form-field-container.card {
+	width: 100%;
+}
+
+
+
+.form-field-container.card label {
+	width: 100%;
+	float: left;
+}
+
+.form-field-container.card .num {
+	width: calc(50% - 0.5rem);
+	float: left;
+}
+.form-field-container.card .exp {
+	width: calc(25% - 0.25rem);
+	float: left;
+	margin-left: 1rem;
+	border-top-right-radius: 0;
+	border-bottom-right-radius: 0;
+}
+.form-field-container.card .cvc {
+	width: calc(25% - 0.25rem);
+	float: right;
+	border-top-left-radius: 0;
+	border-bottom-left-radius: 0;
+}
+
+
+.columns {	
+	width: 100%;
+	display: flex;
+	flex-direction: row;
+}
+
+.main.columns {
+	height: 100vh;
+	overflow: hidden;
+}
+
+.column {
+	width: 50%;
+}
+
+.price-column {
+	width: 33%;
+	background: transparent linear-gradient(233deg, #D500F9 0%, #2962FF 99%) 0% 0% no-repeat padding-box;
+	display: flex;
+	flex-direction: column;
+	justify-content: space-between;
+	color: #fff;
+	padding: 1rem;
+	height: 100%;
+	font: 400 14px/19px 'Open Sans';
+}
+
+.price-name {
+	font: 600 16px/22px 'Open Sans';
+}
+
+.price .val {
+	font: 400 14px/18px 'Days One';
+}
+
+.price .total.val {
+	font: 400 24px/31px 'Days One';;
+}
+
+.price .valxm {
+	font: 400 14px/19px 'Open Sans';
+}
+
+table tr {
+	height: 3rem;
+}
+
+.disclaimer {
+	font: 400 12px/17px 'Open Sans';
+}
+
+.disclaimer a {
+	color: #fff;
+	text-decoration: none;
+}
+
+.wrapper-form {
+	width: 67%;
+	overflow-y: scroll;
+	position: relative;
+	height: 89%;
+}
+
+.button-bar {
+	position: fixed;
+    display: flex;
+    width: 67%;
+    justify-content: space-between;
+	bottom: 0;
+	background-color: #fff;
+	padding: 1rem;
+	margin: 0 -0.5rem;
+}
+
+
+
+.button-bar button {
+	background: transparent linear-gradient(257deg, #D500F9 0%, #2962FF 99%) 0% 0% no-repeat padding-box;
```

Comment:
Codacy found an issue: [Expected "#D500F9" to be "#d500f9" (color-hex-case)](https://app.codacy.com/gh/CTMobi/Wubtitle-Wordpress-Plugin/pullRequest?prid=5886534)